sekoia_automation/connector/metrics.py

```diff
@@ -0,0 +1,33 @@
+from prometheus_client import Counter, Gauge, Histogram
+
+
+class MetricsMixin:
+    _prometheus_namespace = "symphony_module_common"
+
```

Comment:
I believe we usually have an `incoming messages` counter too?
```suggestion
_incoming_events: Counter  = Counter(
    name="collected_messages",
    documentation="Number of messages consumed",
    namespace=prom_namespace,
    labelnames=["intake_key"],
)
```

Reply:
To be honest I did not use it... But I can add if it is useful